src/data/markdown/translated-guides/en/05 Test Types/03 Stress testing.md

```diff
@@ -4,227 +4,104 @@ head_title: 'What is Stress Testing? How to create a Stress Test in k6'
 
-While [load testing](/test-types/load-testing) is primarily concerned with assessing the systems performance,
-the purpose of stress testing is to assess the availability and stability of the system under heavy load.
+The load pattern of a stress test resembles that of an average load test. The main difference is higher load.To account for higher load, the ramp-up period takes longer in proportion to how higher the load must increase. Similarly,  after the test reaches the desired load, it might last for slightly longer  than it would in  the Average-Load test.
 
-## What is stress testing?
+![Overview of a stress test](images/chart-stress-test-overview.png)
 
-Stress Testing is a type of load testing used to determine the limits of the system.
-The purpose of this test is to verify the stability and reliability of the system under **extreme conditions**.
+In some testing conversation, stress tests might also be called rush-hour, surge, or scale tests.
 
-To execute a proper stress test, you need a tool to push the system over its normal operations, to its limits, and _beyond the breaking point_.
+## When to perform a Stress test
 
-You typically want to stress test an API or website to determine:
+The purpose of this test is to verify the stability and reliability of the system under conditions of heavy use.
+Systems may receive higher than usual workloads on unusual moments such as process deadlines, paydays, rush hours, end of workweeks, along with many other isolated behaviors that might combine to create high-load events.
 
-- How your system behaves under extreme conditions.
-- What the maximum capacity of your system is in terms of users or throughput.
-- What is the breaking point of your system and its failure mode.
-- Whether  your system will recover without manual intervention after the stress test is over.
+## Considerations
 
-When stress testing, configure the test to include more concurrent users or generate higher throughput than:
+When you run a stress test, consider the following:
 
-* Your application typically sees.
-* You think your application can handle.
+* **Load should be higher than what the system experiences on average.** 
 
-Note that a stress test doesn't overwhelm the system
-immediately—that's a [spike test](#spike-testing), which is covered in the next section.
+  Some testers might have default targets for stress tests&mdash;say an increase upon average load by 50 or 100 percent&mdash; there's no fixed percentage.
 
-A stress test should be configured in many gradual steps, each step increasing the concurrent load of the system.
+  The load simulated in a Stress test depends on the stressful situations that the system may be subject to. Sometimes this may be just a few percentage points above that average. Other times, it can be 50 to 100% higher, as mentioned above. Even some stressful situations can be twice, triple, or even orders of magnitude higher.
```

Comment:
```suggestion
  The load simulated in a Stress test depends on the stressful situations that the system may be subject to. Sometimes this may be just a few percentage points above that average. Other times, it can be 50 to 100% higher, as mentioned. Some stressful situations can be twice, triple, or even orders of magnitude higher.
```